### PR TITLE
[MDK]: Updated tutorials with latest changes+added some validation Questions

### PR DESCRIPTION
**Pull request by jitendrakansal**
Updated MDK tutorials with latest changes+added some validation Questions

### CODE REVIEW

File: tutorials/cp-mobile-dev-kit-build-client/cp-mobile-dev-kit-build-client.md [hunk 1]
```diff
@@ -224,30 +222,28 @@ Once you've installed these prerequisites, your machine is ready to generate and
...
 
     !![MDK](img-3.2.3.png)
 
 3. Open the `MDKProject.json` file and update it as needed. This file has some build-time configurations such as the application name, version and bundle ID.
 
-    !![MDK](img_3.3.png)
+    !![MDK](img-3.3.png)
 
-    >`AppName`: Provide a name for example: `DemoSampleApp`. This is the name of the application on the home screen of the device.
+    >`AppName`: Provide a name for example: `DemoSampleApp`. It is the name of the application on the home screen of the device and it is also be the name of the folder where the client is created.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-build-client/cp-mobile-dev-kit-build-client.md:241]: "... and it is also be the name..."

Please correct sentence.

File: tutorials/cp-mobile-dev-kit-build-client/cp-mobile-dev-kit-build-client.md [hunk 2]
```diff
@@ -224,30 +222,28 @@ Once you've installed these prerequisites, your machine is ready to generate and
...
-1. In the `MDKClient_SDK` folder, you will find the `template.mdkproject` folder. It is recommended that you copy this folder to another location to use it for future builds and paste it anywhere want and you can rename template to whatever you want but the directory name needs to end in `.mdkproject`.
-
-    !![MDK](img-3.2.1.png)
+1. In the `MDKClient_SDK` folder, you will find the `template.mdkproject` folder.
 
-    The idea of retaining the `.mdkproject` folder is so you can use it again in the future to build additional MDK clients using the same settings using a different version of the MDK SDK.
+    !![MDK](img-3.png)
 
-    >For this tutorial, I have named my `.mdkproject` folder `DemoSampleApp.mdkproject` and put it outside of the MDK Client folder structure. I will refer this name for the rest of this tutorial.
+    It is recommended that you copy this folder to another location so that you can to use it for future builds. Copy and paste it anywhere, and then rename template to `DemoSampleApp.mdkproject`.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-build-client/cp-mobile-dev-kit-build-client.md:229]: "..., and then rename the template..."

File: tutorials/cp-mobile-dev-kit-build-client/cp-mobile-dev-kit-build-client.md [hunk 3]
```diff
@@ -372,15 +368,15 @@ Once you've installed these prerequisites, your machine is ready to generate and
 
     !![MDK](img_5.3.png)
 
-    >To run the MDK client on Android emulator, you need to first create the client for emulator and then use `tns run android --emulator` command. Before trying to launch the client on Android emulator, make sure that you have already configured a virtual device (Android Studio>AVD Manager). Otherwise, you may get error like No emulator image available for device identifier.
+    >To run the MDK client on Android emulator, you need to use `tns run android --emulator` command. Before trying to launch the client on Android emulator, make sure that you have already configured a virtual device (Android Studio>AVD Manager). Otherwise, you may get error like No emulator image available for device identifier.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-build-client/cp-mobile-dev-kit-build-client.md:371]: "...you may get an error..."

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by MichaelCzcz** [comment]
@thecodester This PR contains two new tutorials. Please take over. Thank you.

**Comment by jitendrakansal** [comment]
@MichaelCzcz thanks for your feedback/comments. I have updated and updated the PR.

**Comment by thecodester** [comment]
@jitendrakansal How should we group the 2 new tutorials?

**Comment by jitendrakansal** [comment]
@thecodester I did send you a separate email on this.
Anyway, here are the details:

1.  **cp-mobile-dev-kit-i18n-localization** should be part of [Level up with Mobile Development Kit](https://developers.sap.com/mission.mobile-dev-kit-level-up.html) mission , this tutorial should be added as **2nd last tutorial** in the list. 
image
 Also  please update **mission total time: 3 hr. 20 min**
2. **cp-mobile-dev-kit-cert-auth** should be part of [Brand Your customized App with Mobile Development Kit SDK](https://developers.sap.com/mission.mobile-dev-kit-brand.html) mission as **the last tutorial.**
also please update **mission total time: 1hr. 55 min**

**Pull request merged**